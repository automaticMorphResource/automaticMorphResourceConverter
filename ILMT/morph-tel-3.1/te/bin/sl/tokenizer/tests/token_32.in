సువిశాలమైన ఎత్తైన కొండలు హెచ్చుతగ్గులుండటంచేత అందుకు అనుగుణంగా చుట్టూ ఒకే ప్రాకారాన్ని కాకుండా, ఒకదాని వెనుక మరొకటిగా అనేక ప్రాకారాలను సంపూర్ణ పరిజ్ఞానంతో నిర్మించారు . 
