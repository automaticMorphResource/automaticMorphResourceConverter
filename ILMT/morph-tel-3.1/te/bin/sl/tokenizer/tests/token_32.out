<Sentence id="1">
1	సువిశాలమైన	unk
2	ఎత్తైన	unk
3	కొండలు	unk
4	హెచ్చుతగ్గులుండటంచేత	unk
5	అందుకు	unk
6	అనుగుణంగా	unk
7	చుట్టూ	unk
8	ఒకే	unk
9	ప్రాకారాన్ని	unk
10	కాకుండా	unk
11	,	unk
12	ఒకదాని	unk
13	వెనుక	unk
14	మరొకటిగా	unk
15	అనేక	unk
16	ప్రాకారాలను	unk
17	సంపూర్ణ	unk
18	పరిజ్ఞానంతో	unk
19	నిర్మించారు	unk
20	.	unk
</Sentence>

